javascript.ru/2.5/2.5.html

```diff
@@ -0,0 +1,12 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<meta charset="utf-8" />
+	<title>2.5</title>
+	<script src="2.5.js"></script>
```

Comment:
1. Add `type="text/javascript` to `script` tag.
2. Choose one sorted list of tags in head (check in 2.1). So it will be easily for you in future if all heads have same order of tags. As for me such order as here is good and I would just change `2.1.html` tags title and script.

Reply:
Done in be9ad84a6ac71142969389aa7c435cbb1c87e8d0.